membership/README.md

```diff
@@ -0,0 +1,124 @@
+  - **Links**
+	- https://www.linkedin.com/in/mikeboremi/
+	- https://github.com/mikeyb
+- Phillip O. (@Kryptoniq)
+  - **Strengths:** Token Listing Submissions
+  - **Bio:** 19, Website developer, knows some PHP/MySQL, HTML, CSS and Solidity. Has startup experience and founder of SporChain.
+  - **Links**
+- Michael Razoumovitch (@mira)
+  - **Strengths:** Architectural Design / Software Development
+  - **Bio:**
+  - **Links**
+- Benson Tran (@Flatlines)
+  - **Strengths:** Programming / Discord Community Management / Economics
+  - **Bio:**
+  - **Links**
+    - https://www.linkedin.com/in/benson-tran-51a708150/
+- Rohat Sahin (@rowmagnon)
+  - **Strengths:** Community Management / Development
+  - **Bio:** Webdev experience and project management
+  - **Links**
+- Serina Piencak (@SinGoat)
+  - **Strengths:** Journalism / Writing & Editing / Social Media Management
+  - **Bio:** 20yr old journalism student.  Content writing and editing. Forced into the crypto world by a goat, hodl'ing for my future.
+  - **Links**
+- Kirby Ong (@kirbyong)
+  - **Strengths:** Writing / Community Management / Social Media Management
+  - **Bio:**
+  - **Links**
+- Adam Marsden (@Adam)
+  - **Strengths:** CSS Architecture / UI & UX Design
+  - **Bio:**
+  - **Links**
+- Shachindra Kumar (@shachindra)
+  - **Strengths:** Backend Development / Blockchain Development
+  - **Bio:**
+  - **Links**
+    - https://LinkedIn.com/in/Shachindra92
+- Rally Arceno (@Rally)
+  - **Strengths:** Programming / Software Testing
+  - **Bio:**
+  - **Links**
+    - https://www.linkedin.com/in/rally-arceno-8ba83a6
+- Derek Clauson (@Nidusan Glast)
+  - **Strengths:** Writing / Research Assessment / Marketing
+  - **Bio:**
+  - **Links**
+    - https://www.linkedin.com/in/derek-c-567174153/
+- Francois Devillez (@Splinter)
+  - **Strengths:** Negotiations / Business & Legal Expertise / Networking
+  - **Bio:**
+  - **Links**
+    - https://www.linkedin.com/in/francoisdevillez/
+- Vladmir Agaev (@Vlad)
+  - **Strengths:** Programming / Blockchain Development
+  - **Bio:** Known as Vlad Wulf on YouTube
+  - **Links**
+    - https://www.linkedin.com/in/vladimir-a-16334213b/
+- Andreas Achleithner (@Andreas BTC)
+  - **Strengths:** Engineering Background / Business / Marketing / Community Management
+  - **Bio:** Known as BTCBusinessconsult on YouTube
+  - **Links**
+    - https://www.linkedin.com/in/andreas-achleithner/
+- Imran Khawaja (@imrank1)
+  - **Strengths:** Full Stack Development / Machine Learning / Blockchain Development
+  - **Bio:**
+  - **Links**
+    - https://www.linkedin.com/in/imrank1/
+- Ebru Anik (@ebru)
+  - **Strengths:** Law Graduate
+  - **Bio:** 
+  - **Links**
+    - http://linkedin.com/in/ebru-anik-66769715b
+- James Gold (@James8)
+  - **Strengths:** Programming / Blockchain Development / Ohni CEO
+  - **Bio:** 
```

Comment:
I am student studying computer science. I have advanced knowladge in over 5 programming languages and basic knowledge in over 10, both front end and back end. I successfully (so far) built a crypto currency from the ground up.